Tasks/Common/AzureRmDeploy-common/operations/ContainerBasedDeploymentUtility.ts

```diff
@@ -28,24 +28,29 @@ export class ContainerBasedDeploymentUtility {
         let configFilePath: string = properties["ConfigFilePath"];
         let isMultiContainer: boolean = properties["isMultiContainer"];
         let isLinuxApp: boolean = properties["isLinuxContainerApp"];
-        let updatedConfigFilePath: string;
-        tl.debug("Deploying image " + imageName + " to the webapp " + this._appService.getName());
+        let updatedConfigFilePath: string = configFilePath;
+
+        if(imageName) {
+            tl.debug("Deploying image " + imageName + " to the webapp " + this._appService.getName());
```

Comment:
This statment is correct for single container deployment but for multi-container mention config file name instead.